docs/www/coprocessors.md

```diff
@@ -0,0 +1,122 @@
+# coprocessors
+
+The coprocessor implementation allows you to deploy functional units that operate on the data moving through redpanda logs. The advantage of using the built in coprocessor engine is a drastic improvement in performance, reducing the raft chatter that would otherwise exist if you were to perform transformations off cluster, and it removes any operational complexity that would be introduced by needed to host/manage your transforms on seperate hardware.
+
+For now you can only deploy Javascript functions, however in the future we plan to support all of the WASM supported languages.
+
+## Getting started
+
+To start you'll need to ensure you have `nodejs`, `rpk` and `redpanda` installed on your machine. Then in your `redpanda.yaml` configuration file, you'll need to enable developer and coprocessing mode like so:
+
+```
+redpanda:
+  ...
+  developer_mode: true
+  enable_coproc: true
+  ...
+```
+
+### Creating your first coprocessor
+
+To start, run `rpk wasm generate` in a fresh directory. This will write an example project scaffolding that you can use to get started even quicker.
```

Comment:
```suggestion
To start, run `rpk wasm generate` in a fresh directory. This will write an example project scaffolding that you can use to get started with.
```